tutorial/00_getting_started.ipynb

```diff
@@ -7,6 +7,13 @@
     "# Getting started with `sbi`"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note, you find the original version of this notebook at [https://github.com/mackelab/sbi/tree/main/tutorial](https://github.com/mackelab/sbi/tree/main/tutorial) in the `sbi` repository."
```

Comment:
Why not directly link to the notebook? Link would then be sth like:
`https://github.com/mackelab/sbi/tree/main/tutorial/00_getting_started.ipynb`